src/cryptoadvance/specter/services/swan/templates/swan/oauth2_start.jinja

```diff
@@ -1,5 +1,7 @@
 {% extends "base.jinja" %}
-
+{% block head %}
+<meta http-equiv="refresh" content="30">
```

Comment:
Perhaps we can find a better way after the release for reloading the oauth starting page after linking the account.